ext/storage_sources/s3_store/s3_connection.h

```diff
@@ -34,5 +34,7 @@ class S3Connection {
     const Aws::S3Crt::S3CrtClient _s3CrtClient;
     const std::string _bucketName;
     const std::string _objectPrefix;
+
+    int BucketExists(bool &exists) const;
```

Comment:
np: I think we usually declare the functions before the members. This comes from the fact that you may declare the body of a function here and use a member that is declared below which may look weird.

Reply:
Good point, fixed.